src/components/docPage/index.tsx

```diff
@@ -78,6 +84,16 @@ export function DocPage({
               <div id="main">
                 <CodeContextProvider>{children}</CodeContextProvider>
               </div>
+
+              <div className="grid grid-cols-2 gap-4 not-prose mt-8">
```

Comment:
just a small nit - would use `mt-16` here to allow for more space between content and navigation